modules/contentbox/models/importers/ContentBoxImporter.cfc

```diff
@@ -112,6 +114,7 @@ component accessors=true {
 				overwriteFiles = true
 			);
 			descriptorContents = fileRead( getTempDirectory() & "descriptor.json" );
+			descriptorContents = replaceNoCase( descriptorContents, "null,", "99,", "all" );
```

Comment:
What's this for?

Reply:
When the import runs, some of the values in descriptor.json have priority set to NULL which errors when running execute() since priority is not numeric. Replacing NULL for a number will allow that to run.

Reply:
Ohhh, wondering where did that priority come from.  Can you just set them to 0 instead.

Reply:
Updated and pushed. The ones that had priority null were for emailtemplates, widgets, and medialibrary.